tests/smoke_test.go

```diff
@@ -748,6 +748,12 @@ func TestImportVersionSmoke(t *testing.T) {
 	e.RunCommand("pulumi", "login", "--cloud-url", e.LocalURL())
 	e.RunCommand("pulumi", "new", "random-go", "--yes")
 
+	// Assert the template _does_ use 4.13.0
+	gomod := filepath.Join(projectDir, "go.mod")
+	goMod, err := os.ReadFile(gomod)
+	require.NoError(t, err)
+	assert.Contains(t, string(goMod), "github.com/pulumi/pulumi-random/sdk/v4 v4.13.0")
```

Comment:
This looks like a test that will break eventually, when we upgrade templates.  To make this a little less likely to fail in the future, can we grab the version we want for the random plugin here instead of just asserting it?

Reply:
Yeh we should probably just copy the template locally to this repo